contracts/neutron_interchain_queries/src/contract.rs

```diff
@@ -241,6 +283,19 @@ fn query_recipient_txs(deps: Deps<NeutronQuery>, recipient: String) -> NeutronRe
     Ok(to_binary(&GetRecipientTxsResponse { transfers: txs })?)
 }
 
+pub fn query_cw20_balance(
```

Comment:
if we create a `new_register_cw20_balance_query_msg` in the helpers package, we should move this one to the helpers package as well